test/03-device-state-v2.ts

```diff
@@ -64,22 +63,13 @@ const waitFor = async (fn: () => boolean, timeout: number = 10000) => {
 };
 
 describe('Device State v2', () => {
-	let admin: string;
+	let admin: UserObjectParam;
```

Comment:
BTW, I really like this pattern here @Page-, where we declare the test case "state" as *typed* local variables instead of stuffing everything untyped in `this` or `ctx`. As much as I don't (personally) like keeping state between tests, I don't see us moving away from doing so (at least in the short term), so I think we should adopt this pattern in other repos as well. This at least gives some visibility over what exactly is local state, and also allows refactoring because of the static typing, which is a big improvement over what we currently have.

Reply:
The `this` pattern started in coffeescript where it makes a lot more sense btw and it's still in the typescript version of most converted tests because the focus is getting to typescript ASAP to avoid writing new code in coffeescript and getting benefits for most of the code.  Long-term it should all be standardized with proper typing but it'll be a much more gradual process

Reply:
Yeah, I know, I didn't mean to suggest we're not making progress. This was more of a heads up to you to keep an eye out in review in order to suggest this as an improvement if needed.